test/javascripts/components/d-editor-test.js.es6

```diff
@@ -193,6 +193,23 @@ testCase('link modal (simple link)', function(assert) {
   });
 });
 
+testCase('link modal (simple link) with selected text', function(assert, textarea) {
+  this.set('value', "hello world.");
```

Comment:
If you are using the `testCase` helper it should default to `hello world.`. Also you shouldn't need an `andThen` to set the selectionStart and End, other tests don't  need it!